webpack.config.js

```diff
@@ -13,13 +14,27 @@ const PATHS = {
     serverbuild: path.resolve(__dirname, 'src/encoded/static/build-server'),
     fonts: path.resolve(__dirname, 'src/encoded/static/font'),
     images: path.resolve(__dirname, 'src/encoded/static/img'),
+    ckeditor_source: path.resolve(__dirname, "node_modules/ckeditor4"),
+    ckeditor_dest: path.resolve(__dirname, "src/encoded/static/build/ckeditor"),
 };
 
 const plugins = [];
 
 // To get auth0 v11 to build correctly:
 // https://github.com/felixge/node-formidable/issues/337#issuecomment-183388869
 plugins.push(new webpack.DefinePlugin({ 'global.GENTLY': false }));
+plugins.push(
+    new CopyPlugin({
+        patterns: [
+            {
+                // Adapted from https://github.com/ckeditor/ckeditor4-webpack-template/blob/master/webpack.config.js
```

Comment:
Thanks for the link; that helps a lot. I’m not very good at configuring webpack.

Reply:
Me neither apparently, took a while mucking about with webpack to get it to work.